backend/src/test/scala/cromwell/backend/BackendSpec.scala

```diff
@@ -95,7 +105,8 @@ trait BackendSpec extends ScalaFutures with Matchers {
     BackendJobDescriptor(workflowDescriptor, firstJobDescriptorKey(workflowDescriptor), Map.empty, inputs)
   }
 
-  def createRuntimeAttributes(wdlSource: WdlSource, runtimeAttributes: String = "") = {
+  @deprecated("This is an unneccessarily circuitous way of generating a Map[String, WdlValue]...", "Sep 2 2016")
```

Comment:
When I had a look, I thought they were all calling `.head` on the result. But I'll double check